plugins/woocommerce-blocks/assets/js/settings/shared/default-fields.ts

```diff
@@ -100,6 +100,7 @@ export interface CoreAddress {
 	state: string;
 	postcode: string;
 	phone: string;
+	email?: string;
```

Comment:
What did this fix? Its not common to both address types.

Reply:
I can't recall, let's see if type fixing fixes that.